contrib/opencensus-context/examples/span.py

```diff
@@ -0,0 +1,26 @@
+from opencensus.common.runtime_context import RuntimeContext
+
+RuntimeContext.register_slot('current_span', None)
```

Comment:
This is to demonstrate how we can leverage RuntimeContext to build trace/execution_context (tracer, current_span, attrs), tags/execution_context, W3C CorrelationContext and TraceContext. So all of them will benefit from the generic context in async and explicit threading (customer explicitly created threads and propagate the context manually) scenario.

In this way Dynatrace and also introduce an extension/exporter which registers its own context, without having to pollute span_context (e.g. if we want to propagate something within the process, but don't want to serialize/send across the wire).

Reply:
Would you expect e.g. the correlation context package to have another single module-level context object? Like:

```correlation_context = RuntimeContext.register_slot('correlation_context', dict)```

This makes sense to me, but thinking about the API: as a user, why use `RuntimeContext` instead of `contextvars.Context` directly?

Reply:
Good idea, I will definitely try this approach.

Regarding letting user to use `contextvars.Context`, that is our direction - we don't want to introduce another context if the runtime already has it.

For now, given we need to support Python 2.7/3.4-3.6, probably we need to backport? I checked contextvars and didn't see a good way to backport as the logic was implemented as a native module (in C rather than Python code).

Reply:
https://pypi.org/project/contextvars/ for 3.6 we have contextvars looks like.